pcl_cpu_ndt:
  ros__parameters:
    x_bound: 15.0
    y_bound: 15.0
    z_bound: 5.0
    downsample_voxel_size: 0.1
    max_icp_iter: 50
    transformation_epsilon: 0.01
    step_size: 0.25
    resolution: 1.0
    max_correspondence_distance: 3.0
    debug: True
